src/libs/portfolio/interfaces.ts

```diff
@@ -229,3 +240,7 @@ export type FormattedPendingAmounts = Omit<PendingAmounts, 'pendingBalance'> & {
   pendingToBeSignedFormatted?: string
   pendingToBeConfirmedFormatted?: string
 }
+
+export type FirstCashbackConfettiStatusByAccount = {
+  [key: string]: boolean
+}
```

Comment:
Technically a better place to write my comment from https://github.com/AmbireTech/ambire-common/pull/1135/files#r1939732041